e2e/cypress/tests/support/ui/global_header.js

```diff
@@ -27,7 +27,9 @@ Cypress.Commands.add('uiOpenProductMenu', (item = '') => {
 });
 
 Cypress.Commands.add('uiGetSetStatusButton', () => {
-    return cy.findByRole('button', {name: 'set status'}).should('be.visible');
+    return cy.get('.status-wrapper > .Avatar').should('be.visible');
```

Comment:
why did we had to do this way? did the semantics change if yes should we not try to use testing principles :P